lib/cluster.js

```diff
@@ -261,7 +261,7 @@ function masterInit() {
     assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
            'Bad cluster.schedulingPolicy: ' + schedulingPolicy);
 
-    var hasDebugArg = process.execArgv.some(function(argv) {
+    var hasDebugArg = process.execArgv.some(function hasDebugTest(argv) {
```

Comment:
doesn't throw, making it an anon es6 function would seem to be more consistent with node practice, wouldn't it?